src/test/resources/features/dcu/DCUSearch.feature

```diff
@@ -80,22 +57,11 @@ Feature: DCU Search
     | TRO        | TRO + DTEN          |
     | MIN        | All DCU Case Types  |
 
-  @SearchByCaseType @SearchByTopic
-  Scenario: User should be able to search by multiple parameters
-    Given I am logged into "CS" as user "DCU_USER"
-    And I create a "MIN" case with "Animal alternatives (3Rs)" as the primary topic
-    And I navigate to the "search" page
-    When I enter "MIN" into the "Case Type" search field in the "DCU" search configuration
-    And I enter "Animal alternatives (3Rs)" into the "Topic" search field in the "DCU" search configuration
-    And I click the search button on the search page
-    Then I check that the search results have the correct "Case Type"
-    And I check that the search results have the correct "Topic"
-
   @SearchByDateReceived
   Scenario Outline: No cases should be displayed if a user searches for a criteria that would contain no cases
     Given I am logged into "CS" as user "DCU_USER"
     And I navigate to the "search" page
-    When I enter "<infoValue>" into the "<infoType>" search field in the "DCU" search configuration
+    When I enter "<infoValue>" into the "<infoType>" search field
     And I click the search button on the search page
     Then 0 cases should be displayed
     Examples:
```

Comment:
do we  really need this scenario? and if we do, do we need 3 examples for it? 1 seems adequate